src/test_take_screenshots.ts

```diff
@@ -56,10 +56,25 @@ describe("Agama screenshots", function () {
   setInitialRootPassword(options.rootPassword);
 
   it("should take overview page screenshot", async function () {
-    await page.locator("h3::-p-text('Overview')").wait();
+    await page.locator("main ::-p-text('Overview')").wait();
```

Comment:
Agama 11 changed `<h3>` to `<h2>`. Let's make it more generic and use the parent `<main>` instead.